pkg/features/kube_features.go

```diff
@@ -466,6 +466,7 @@ const (
 
 	// owner: @robscott @freehan
 	// alpha: v1.16
+	// ga: v1.21
```

Comment:
Apparently I missed adding a comment for beta 🤦. Do you mind adding a `beta: v1.18` line here?

Reply:
Good catch! Added in `beta: 1.18` in the newest update